<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertDialog</class>
 <widget class="QDialog" name="AlertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>졸음운전 경고</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(40, 40, 40, 220);
border-radius: 15px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="iconLabel">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgba(255, 82, 82, 180);
border-radius: 10px;
padding: 15px;</string>
     </property>
     <property name="text">
      <string>⚠️ 졸음 운전이 감지되었습니다!
안전한 곳에 정차 후 휴식을 취하세요.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #e53935;
color: white;
border-radius: 8px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
